src/GraphQL/Introspection/__Directive.cs

```diff
@@ -48,7 +48,7 @@ public class __DirectiveLocation : EnumerationGraphType<DirectiveLocation>
     {
         public __DirectiveLocation()
         {
-            Name = nameof(__DirectiveLocation);
+            SetName(nameof(__DirectiveLocation), validate: false);
```

Comment:
`EnumerationGraphType` has already set `Name` in ctor to `DirectiveLocation` so need to set it again to `__DirectiveLocation` without validation. The same for `__TypeKind`.